arangod/RocksDBEngine/RocksDBMetadata.cpp

```diff
@@ -468,6 +466,8 @@ Result RocksDBMetadata::serializeMeta(rocksdb::WriteBatch& batch,
         << "collection with objectId '" << rcoll->objectId() << "'";
```

Comment:
It looks to me as if this `else` is now executed for a collection which does not have `useSyncByRevision` which will not really work, or am I mistaken?